tf2_ros/test/time_reset_test.cpp

```diff
@@ -27,79 +27,85 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <chrono>
 #include <gtest/gtest.h>
 #include <tf2_ros/transform_broadcaster.h>
 #include <tf2_ros/transform_listener.h>
-#include <rosgraph_msgs/Clock.h>
+#include <rosgraph_msgs/msg/clock.hpp>
+#include <builtin_interfaces/msg/time.hpp>
+#include <rclcpp/rclcpp.hpp>
 
-using namespace tf2;
+void spin_for_a_second(std::shared_ptr<rclcpp::Node>& node)
+{
+  rclcpp::Rate r(10);
+  rclcpp::spin_some(node);
+  for (uint8_t i = 0; i < 10; ++i)
```

Comment:
This really doesn't matter at all here, but for future reference an `int` loop counter will be more efficient than a `uint8_t`.